package/src/components/ImageGallery/ImageGallery.tsx

```diff
@@ -179,35 +146,20 @@ export const ImageGallery = <
     },
   } = useTheme();
   const [gridPhotos, setGridPhotos] = useState<Photo<StreamChatGenerics>[]>([]);
-  const { overlay, translucentStatusBar } = useOverlayContext();
+  const { overlay } = useOverlayContext();
   const { messages, selectedMessage, setSelectedMessage } =
     useImageGalleryContext<StreamChatGenerics>();
 
   const { vh, vw } = useViewport();
 
-  const measuredScreenHeight = vh(100);
```

Comment:
are you sure that window needs to be used here and not screen? confusing.. but if you are sure then fine, because previously it is screen

Reply:
![Uploading Screenshot_1713521047.png…]()
If Screen is used

Reply:
If screen is used:
![Screenshot_1713521047](https://github.com/GetStream/stream-chat-react-native/assets/39884168/e7ac95d6-071e-4616-979b-b694e1fd0a5b)

If window is used:
![Screenshot_1713521106](https://github.com/GetStream/stream-chat-react-native/assets/39884168/d09d5a12-e745-4ae7-81a8-0db9329d82d8)

That is why I think window is better